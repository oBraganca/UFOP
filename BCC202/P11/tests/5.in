1 Isabella 10 15 9 1938
1 Alice 3 2 2 2001
1 Miguel 7 26 4 1981
1 Alice 9 25 7 1998
1 Bernardo 8 26 3 1937
2
1 Miguel 8 19 9 1934
1 Helena 7 25 5 1933
1 Isabella 9 17 7 1934
1 Arthur 2 25 12 2003
1 Bernardo 3 12 12 2010
1 Helena 8 22 7 1944
1 Miguel 8 23 10 1995
1 Helena 8 21 10 1951
1 Sophia 7 17 6 1957
1 Arthur 8 28 8 2003
2
1 Miguel 3 8 9 1968
1 Miguel 1 22 7 2003
1 Bernardo 10 21 10 1973
1 Alice 8 28 3 2001
1 Sophia 2 18 2 1962
1 Arthur 2 29 1 1962
1 Davi 10 22 9 1949
1 Isabella 6 14 3 2007
1 Pedro 5 5 7 2015
1 Isabella 5 8 12 1941
2
1 Alice 3 10 3 1942
1 Arthur 2 29 6 1973
1 Pedro 9 28 9 1942
1 Pedro 9 1 3 1941
1 Isabella 1 21 5 1986
1 Gabriel 1 7 4 1970
1 Helena 10 20 3 1950
1 Pedro 4 10 1 1986
1 Gabriel 9 12 11 2003
1 Bernardo 5 1 10 1964
1 Miguel 3 21 7 1993
1 Miguel 10 2 12 1958
1 Alice 9 3 6 1949
1 Helena 6 5 9 1960
1 Davi 9 23 2 2015
1 Miguel 4 16 6 1947
1 Alice 8 24 8 1935
1 Arthur 8 4 10 2007
1 Bernardo 3 20 3 1996
1 Helena 4 7 11 1967
2
1 Pedro 2 24 4 1984
1 Miguel 9 24 5 2020
1 Pedro 2 13 12 1961
1 Pedro 7 11 11 2014
1 Bernardo 10 27 8 1985
2
3
