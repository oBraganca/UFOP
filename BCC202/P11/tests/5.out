10 | 15/09/1938 | Isabella
09 | 17/07/1934 | Isabella
10 | 22/09/1949 | Davi
10 | 20/03/1950 | Helena
10 | 02/12/1958 | Miguel
10 | 21/10/1973 | Bernardo
10 | 27/08/1985 | Bernardo
09 | 01/03/1941 | Pedro
09 | 28/09/1942 | Pedro
09 | 03/06/1949 | Alice
09 | 25/07/1998 | Alice
09 | 12/11/2003 | Gabriel
09 | 23/02/2015 | Davi
09 | 24/05/2020 | Miguel
08 | 19/09/1934 | Miguel
08 | 24/08/1935 | Alice
08 | 26/03/1937 | Bernardo
08 | 22/07/1944 | Helena
08 | 21/10/1951 | Helena
08 | 23/10/1995 | Miguel
08 | 28/03/2001 | Alice
08 | 28/08/2003 | Arthur
08 | 04/10/2007 | Arthur
07 | 25/05/1933 | Helena
07 | 17/06/1957 | Sophia
07 | 26/04/1981 | Miguel
07 | 11/11/2014 | Pedro
06 | 05/09/1960 | Helena
06 | 14/03/2007 | Isabella
05 | 08/12/1941 | Isabella
05 | 01/10/1964 | Bernardo
05 | 05/07/2015 | Pedro
04 | 16/06/1947 | Miguel
04 | 07/11/1967 | Helena
04 | 10/01/1986 | Pedro
03 | 10/03/1942 | Alice
03 | 08/09/1968 | Miguel
03 | 21/07/1993 | Miguel
03 | 20/03/1996 | Bernardo
03 | 02/02/2001 | Alice
03 | 12/12/2010 | Bernardo
02 | 13/12/1961 | Pedro
02 | 29/01/1962 | Arthur
02 | 18/02/1962 | Sophia
02 | 29/06/1973 | Arthur
02 | 24/04/1984 | Pedro
02 | 25/12/2003 | Arthur
01 | 07/04/1970 | Gabriel
01 | 21/05/1986 | Isabella
01 | 22/07/2003 | Miguel
